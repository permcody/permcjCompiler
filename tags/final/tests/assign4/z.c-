//
// Randomly generated C- code
//
bool w(void)
{
    void w;
    bool v;
    while (-v)
	if (true + v) {
	    bool v;
	    while (-11 - 3) {
	    }
	}
	else;
    if (u = !y) {
	bool z;
	y()>--x;
	true;
    }
    else if (w = 0) { {
    }
    }
    else if (u())
	if (x + 7 - v)
	    v;
    x = u%true/w;
}
