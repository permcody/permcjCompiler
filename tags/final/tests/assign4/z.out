ERROR lineno(6): w cannot be of type void.
ERROR lineno(8): Unary - requires things of type int but was given type bool.
ERROR lineno(9): + requires things of type int but lhs is of type bool.
ERROR lineno(9): + requires things of type int but rhs is of type bool.
ERROR lineno(11): Expecting boolean test condition.
ERROR lineno(15): Symbol u is not defined.
ERROR lineno(15): Symbol y is not defined.
ERROR lineno(17): Symbol y is not defined.
ERROR lineno(17): Symbol x is not defined.
ERROR lineno(20): Cannot use function w as a simple variable.
ERROR lineno(23): Symbol u is not defined.
ERROR lineno(24): Symbol x is not defined.
ERROR lineno(26): Symbol x is not defined.
ERROR lineno(26): Symbol u is not defined.
ERROR lineno(26): Cannot use function w as a simple variable.
Number of errors: 15
Number of warnings: 0
